app/controllers/admin/videos_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Admin::VideosController < ApplicationController
+  before_action :require_sign_in
+  before_action :require_admin
+
+  def new
+    @video = Video.new
+  end
+
+  def create
+    @video = Video.new(video_params)
+
+    handel_url
```

Comment:
Because its user input and its good habit to not change those directly. Change the data you need as output from it not the source. Also this is more of a presentation layer topic where you could eventually need the `http://`, more about this was in course 2.
Hope this helps.